Tests/SmokeDynamoDBTests/DynamoDBTableHistoricalItemExtensionsTests.swift

```diff
@@ -338,205 +339,213 @@ class DynamoDBHistoricalClientTests: XCTestCase {
-        
+
         do {
             try table.clobberItemWithHistoricalRowSync(primaryItemProvider: testPrimaryItemProvider, historicalItemProvider: testHistoricalItemProvider, withRetries: 9)
-            
+
             XCTFail("Expected error not thrown.")
         } catch SmokeDynamoDBError.concurrencyError(_) {
             // Success
         } catch {
             return XCTFail("Unexpected exception: \(error)")
         }
-       
+
     }
-    
+
     func testClobberItemFailureAsync() throws {
-        
+
         let wrappedTable = InMemoryDynamoDBTable()
         let table = SimulateConcurrencyDynamoDBTable(wrappedDynamoDBTable: wrappedTable,
                                                      simulateConcurrencyModifications: 12)
-        
+
         var isCompleted = false
         try table.clobberItemWithHistoricalRowAsync(primaryItemProvider: testPrimaryItemProvider, historicalItemProvider: testHistoricalItemProvider, withRetries: 9) { error in
             guard let theError = error, case SmokeDynamoDBError.concurrencyError = theError else {
                 return XCTFail("Expected error not thrown")
             }
-            
+
             isCompleted = true
         }
-        
+
         XCTAssertTrue(isCompleted)
     }
-    
+
     private func conditionalUpdatePrimaryItemProvider(existingItem: DatabaseRowType) throws -> DatabaseRowType {
         let rowVersion = existingItem.rowStatus.rowVersion
         let dPayload = TestTypeA(firstly: "firstly_\(rowVersion)", secondly: "secondly_\(rowVersion)")
-    
+
         return try existingItem.createUpdatedRowWithItemVersion(
             withValue: dPayload,
             conditionalStatusVersion: nil)
     }
-    
+
     private let historicalCompositePrimaryKey = StandardCompositePrimaryKey(partitionKey: "historicalPartitionKey",
                                                                                    sortKey: "historicalSortKey")
     private func conditionalUpdateHistoricalItemProvider(updatedItem: DatabaseRowType) -> DatabaseRowType {
         // create an item for the history partition
         return TypedDatabaseItem.newItem(withKey: historicalCompositePrimaryKey,
                                          andValue: updatedItem.rowValue)
     }
-    
+
     func testConditionallyUpdateItemWithHistoricalRowSync() throws {
         let table = InMemoryDynamoDBTable()
-        
+
         let databaseItem = testPrimaryItemProvider(nil)
         try table.insertItemSync(databaseItem)
-        
-        try table.conditionallyUpdateItemWithHistoricalRowSync(
+
+        let inserted = try table.conditionallyUpdateItemWithHistoricalRowSync(
             compositePrimaryKey: dKey,
             primaryItemProvider: conditionalUpdatePrimaryItemProvider,
             historicalItemProvider: conditionalUpdateHistoricalItemProvider)
-        
-        let inserted: DatabaseRowType = (try table.getItemSync(forKey: databaseItem.compositePrimaryKey))!
```

Comment:
I'd probably keep this line and compare it with what is returned from `conditionallyUpdateItemWithHistoricalRowSync`, making sure they are the same.

Reply:
Done.